roles/cuda_toolkit_install/tasks/install.yml

```diff
@@ -1,38 +1,39 @@
-- name: Install cuda-keyring for Debian
-  become: true
-  ansible.builtin.apt:
-    deb: '{{ cuda_keyring_url_base }}/{{ cuda_keyring_file_name }}'
-  when: hostvars[inventory_hostname].ansible_os_family == "Debian"

```

Comment:
It may be better to separate `debian.yml` and `suse.yml` tasks rather than checking every time.

Reply:
Done.